src/components/NavMenu/NavMenu.sass

```diff
@@ -0,0 +1,9 @@
+@import '@/styles/variables'
+
+.nav
+  display: flex
+  align-items: center
+  color: $main-green
+
+.nav li:not(:last-child) 
+  margin-right: 33px
```

Comment:
What about flex gap instead of margin-right?

Reply:
It was an issue with a classname 'nav'. This class has his initial styles, so I renamed it and now the gap works as it should be. Because before renaming gap also added space before first and after the last element in the list. Now it is ok.